lifetimes/estimation.py

```diff
@@ -278,6 +298,7 @@ def fit(self, frequency, monetary_value, iterative_fitting=4, initial_params=Non
                  This model is not very stable so we suggest >10 for best estimates evaluation.
             initial_params: set initial params for the iterative fitter.
             verbose: set to true to print out convergence diagnostics.
+            tol: tolerance for termination of the function minimization process.
```

Comment:
Have you considered passing some of these arguments to `__init__` rather than to `fit`? That'd be more consistent with sklearn conventions.